src/index.html

```diff
@@ -17,16 +17,44 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="get"
+      class="search-bar search bar--big"
```

Comment:
The class attribute should not contain spaces. Consider using a single class name or using hyphens/underscores to separate words.